src/providers/bitcoin/BitcoinLedgerProvider.js

```diff
@@ -161,6 +161,40 @@ export default class BitcoinLedgerProvider extends LedgerProvider {
     }))
   }
 
+  async getWalletInfo (numAddressPerCall = 10, from = {}) {
+    let addressIndex = from.index || 0
+    let unusedAddress = false
+    let usedAddresses = []
+    let balance = 0
+
+    while (!unusedAddress) {
+      let addresses = await this.getAddresses(addressIndex, numAddressPerCall)
+      const addressList = addresses.map(addr => addr.address)
+
+      const addressDeltas = await this.getMethod('getAddressDeltas')(addressList)
+
+      addressDeltas.forEach((delta) => {
+        const addressIndex = addresses.findIndex(address => address.address === delta.address)
+        if (addresses[addressIndex].balance === undefined) { addresses[addressIndex].balance = 0 }
+        addresses[addressIndex].balance += delta.satoshis
+        balance += delta.satoshis
+      })
+
+      addresses.forEach((address) => {
+        if (!unusedAddress) {
+          if (address.balance === undefined) {
```

Comment:
@mattBlackDesign is this right? Are we saying that an unused address is an address that does not have a balance? This doesn't seem right because you could have any number of addresses with 0 balance at the start of your ledger, (i know i do) before you hit one with a balance and as I understand here, unused address would come out wrong and potentially not calculate correct balance too.

In the other method we have used unspents or bitcore's `received` property.